api/schema/application_aws_resources_resource.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Illumio, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package schema
+
+import (
+	resource_schema "github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+var (
+	applicationAwsResourcesResource = Resource{
+		TypeName: "application_aws_resources",
+		Schema: resource_schema.Schema{
+			Version:     1,
+			Description: "Resources associated with an application in the Illumio CloudSecure platform.",
+			Attributes: map[string]resource_schema.Attribute{
+				IDFieldName: idAttribute,
+				"application_id": resource_schema.StringAttribute{
+					Description: "ID of the application.",
+					Required:    true,
+				},
+				"account_id": resource_schema.StringAttribute{
+					Description: "ID of the AWS account.",
+					Required:    true,
+				},
```

Comment:
1. Expand the description to clarify that all the AWS resources below belong to this account.
1. This may need to be immutable (???).
```suggestion
				"account_id": StringResourceAttributeWithMode{
					StringAttribute: resource_schema.StringAttribute{
						Description: "ID of the AWS account the AWS resources belong to.",
						Required:    true,
						PlanModifiers: []planmodifier.String{
							stringplanmodifier.RequiresReplace(),
						},
					},
					attributeWithMode: attributeWithMode{
						Mode: ImmutableAttributeMode,
					},	```